app-modules/prospect/src/Observers/ProspectObserver.php

```diff
@@ -53,11 +53,11 @@ public function creating(Prospect $prospect): void
 
     public function created(): void
     {
-        Cache::tags('prospects')->flush();
+        Cache::tags('{{prospects}}')->flush();
     }
 
     public function deleted(): void
     {
-        Cache::tags('prospects')->flush();
+        Cache::tags('{{prospects}}')->flush();
```

Comment:
This should only have one set of `{}` around this.

In the example you are copying from, it only has the double `{}` because it is within a string with a variable in it and String interpolation.